sklearn/manifold/t_sne.py

```diff
@@ -507,8 +507,8 @@ class TSNE(BaseEstimator):
         The perplexity is related to the number of nearest neighbors that
         is used in other manifold learning algorithms. Larger datasets
         usually require a larger perplexity. Consider selecting a value
-        between 5 and 50. The choice is not extremely critical since t-SNE
-        is quite insensitive to this parameter.
+        between 5 and 50. While the choice is not extremely critical, results
+        have been shown to change considerably across perplexity values.
```

Comment:
That sounds a bit contradictory. Why not simply remove the entire sentence?

Reply:
I think it is important for the user to know that results can change considerably across perplexity values. I could either remove the sentence or do something like:

Reply:
+1 for your proposition